### PR TITLE
Created rbe.bazelrc and imported toolchain configs for RBE.

### PR DESCRIPTION
**Pull request by eytankidron**
@fejta 
Creating a bazelrc for RBE and adding toolchain configs to WORKSPACE.

### CODE REVIEW

**Comment by fejta** [comment]
Please run `hack/update-bazel.sh`

File: rbe.bazelrc [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+# Copyright 2016 The Bazel Authors. All rights reserved.
```
Comment (fejta) [inline rbe.bazelrc]: 2019
Also please assign the copyright to the correct owner (Kubernetes)

File: rbe.bazelrc [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
```
Comment (fejta) [inline rbe.bazelrc]: Please delete this line. Run `hack/verify_boilerplate.py` to ensure this is all correct

File: rbe.bazelrc (hunk 1 above)
Comment (fejta) [reply rbe.bazelrc]: Actually just drop the copyright stanza, we don't need it for config.

Comment (fejta) [reply rbe.bazelrc]: Please address this when you get the chance, then we can merge (drop the copyright)

Comment (eytankidron) [reply rbe.bazelrc]: Done. Removed the copyright stanza.

**Comment by fejta** [comment]
Congrats on your first PR :)

**Comment by fejta** [comment]
https://github.com/kubernetes/test-infra/issues/12282

**Pull request merged**